docs/DeveloperGuide.md

```diff
@@ -172,6 +172,32 @@ Step 5. During the execution of the `AddProjectCommand`, selected `Employee` obj
 
 Step 6. A `CommandResult` is produced based on whether the execution was a success or not and returned to the `LogicManager`. 
 
+### Prioritise Projects Feature
+
+Upon creation of a new `Project`, a `ProjectPriority` can be assigned to that `Project` using the `PriorityP` command. 
+
```

Comment:
Perhaps it should be `priorityP` command instead of `PriorityP`?

Reply:
Yep, good catch. I will try to fix that soon in another PR.

Reply:
@Chandan8186 I have changed this in PR #130.